development/mimir-microservices-mode/config/mimir.yaml

```diff
@@ -198,3 +198,6 @@ limits:
 
 runtime_config:
   file: ./config/runtime.yaml
+
+server:
+  log_level: error
```

Comment:
I think this might be annoying for people, lets not do it. We can turn it on on demand.